### ISSUE TITLE
[NBug] InvalidArgument=Hodnota 0 není platnou hodnotou pro argumen...

### ISSUE THREAD

**Issue by tttpapi** [gitextensions/gitextensions#7341]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Current behaviour
Crash program.

## Expected behaviour
Do not crash the program.

## Steps to reproduce

Copy paste a branch number into branches input which is not in local git and click to show branches.

## Error Details
```
System.ArgumentOutOfRangeException: InvalidArgument=Hodnota 0 není platnou hodnotou pro argument index.
Název parametru: index
   v System.Windows.Forms.ComboBox.ObjectCollection.get_Item(Int32 index)
   v System.Windows.Forms.ComboBox.get_Text()
   v System.Windows.Forms.ComboBox.WmReflectCommand(Message& m)
   v System.Windows.Forms.ComboBox.WndProc(Message& m)
   v System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4018.0
- DPI 96dpi (no scaling)

**Comment by tttpapi** [comment]
Reopen this bug please.

**Comment by gerhardol** [comment]
Duplicate of #6680

Corrected in #7271, please try build for master

**Comment by pmiossec** [comment]
Fixed by #7271

**Comment by tttpapi** [comment]
I have the latest version. There are no updates available. So the fix is not released yet or it is still there.
